icoscp/stilt/stiltobj.py

```diff
@@ -122,7 +122,7 @@ def get_ts(self, start_date, end_date, hours=[], columns=''):
         columns : TYPE, optional
             Valid entries are "default", "co2", "co", "rn", "wind", "latlon", "all"
             default (or empty) will return
-            ["isodate","co2.stilt","co2.fuel","co2.bio", "co2.background"]
+            ["isodate","co2.stilt","co2.fuel","co2.bio","co2.cement","co2.background"]
```

Comment:
@altix the order should probably be `["isodate","co2.stilt","co2.bio","co2.fuel","co2.cement","co2.background"]` since the `co2.fuel` is moved in front of `co2.bio`.